### ISSUE TITLE
Tusk menu won't open after adding a database on Firefox 62 on Linux

### ISSUE THREAD

**Issue by ticpu** [subdavis/Tusk#240]
### This issue is a 

bug

### Steps to reproduce - current behaviour
1. Start a new instance of a fresh Firefox 62 on Linux.
1. Install Tusk.
1. Click Tusk menu to add a database.
1. Open a local, link or gdrive database.
1. Click Tusk menu.
1. The menu appears and instantly disappear.

#### Additional remarks

1. I can see the correct menu for the 50 or so milliseconds it appears.
1. It works fine on Chromium.

#### Environment 

**Operating System:** Ubuntu Bionic

**Browser:** Mozilla Firefox 62 and 63

**Storage:** Local / DAV / Link / Google Drive.

**Comment by ticpu** [comment]
OK so this was absolutely unexpected. I had been searching for a bit and after trying on a VM and another computer it worked so I guessed it was an issue on mine.
Tried:
1. Deleting $HOME/.mozilla
1. Disabling AppArmor.
1. Uninstalling Firefox and running from fresh .tar.bz2 from Mozilla.
1. Starting with env -i and only setting $DISPLAY, this worked.

At this moment I tried isolating each environment variables and found out the culprit was: GTK_IM_MODULE="ibus"

After running unset GTK_IM_MODULES, everything works as expected. I also had an issue which I thought was caused by another add-on I was looking to replace that caused right clicking on password fields to not show the context menu.

In the end, this was good since it caused me to try Tusk which seems to work really well and be much more modern than my older password manager.
